enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx

```diff
@@ -37,6 +38,18 @@ export const PublicComponentStylesWrapper = styled.div`
     display: inline;
   }
 `;
+
+export const PublicComponentStylesWrapper = (
+  props: React.ComponentProps<"div">,
+) => {
+  return (
+    <PublicComponentStylesWrapperInner
+      {...props}
+      // eslint-disable-next-line react/prop-types -- className is in div props :shrugs:
+      className={`mb-wrapper ${props.className}`}
```

Comment:
we could be a little cleaner with `className` by using `cx` from `className` which should handle spacing automatically when `props.className` is undefined.